code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm

```diff
@@ -260,56 +255,58 @@
 	description = "Makes you nearly impervious to stuns and grants a stamina regeneration buff, but you will be a nearly uncontrollable tramp-bearded raving lunatic."
 	reagent_state = LIQUID
 	color = "#60A584" // rgb: 96, 165, 132
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
```

Comment:
Could you not use initial(metabolization_rate) is your multiplier?
So if someone wants to change the metabolizationrate, they only need to change the value at the definitions of the reagent?